### DISCUSSION TITLE
Video support

### DISCUSSION THREAD

**Discussion by AcnoSaga** [singerdmx/flutter-quill#277] [Feature Request]
So, basically, how about we add video picking support in the editor?

**Comment by dwuidcfns** [comment]
Do you mean that the video format is supported? If you need a video editing software that supports many input formats, I recommend you to use TunesKit AceMovi. Although there are many video editing program, relatively speaking, this tool supports a lot more input formats than others.
![670CBB3F-4E67-4fe5-8374-1FD8A03DCB6E](https://user-images.githubusercontent.com/87404766/125554531-f597acf8-31cf-472c-9498-c8b65e3c42b9.png)
